rzup/src/lib.rs

```diff
@@ -11,64 +11,378 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+#[cfg(feature = "cli")]
 pub mod cli;
```

Comment:
Why have this `cli` module in the library at all? Could it just go in the binary part?